zilencer/views.py

```diff
@@ -672,6 +676,71 @@ def update_remote_realm_data_for_server(
     RemoteRealmAuditLog.objects.bulk_create(remote_realm_audit_logs)
 
 
+def get_billable_realm_uuids(realms: List[RealmDataForAnalytics]) -> List[UUID]:
+    billable_realm_uuids = []
+    for realm in realms:
+        # TODO: Remove the `zulipinternal` string_id check once no server is on 8.0-beta.
+        if (
+            realm.is_system_bot_realm
+            or realm.host.startswith("zulipinternal.")
+            or realm.host.startswith("analytics.")
+        ):
+            continue
+        billable_realm_uuids.append(realm.uuid)
+
+    return billable_realm_uuids
+
+
+@transaction.atomic
+def handle_customer_migration_from_server_to_realms(
```

Comment:
This function should write to the audit log when it does its thing. Maybe we can make a `REMOTE_PLAN_TRANSFERRED_SERVER_TO_REALM` event type that we write for both the audit logs?

Can be a follow-up.

Reply:
Done.